From 3c09cd3801d1200a11e03a387cd4844591b7850d Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Tue, 5 Aug 2014 17:02:25 +0200
Subject: [PATCH] configure.ac: enable GNU_SOURCE for stat64/statfs64

Use AC_USE_SYSTEM_EXTENSIONS to enable GNU_SOURCE, which is needed
for:
 - stat64 in utils/exportfs/exportfs.c
 - statfs64 in utils/mountd/cache.c

Signed-off-by: Natanael Copa <ncopa@alpinelinux.org>
---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index 408f4c8..b9682ed 100644
--- a/configure.ac
+++ b/configure.ac
@@ -8,6 +8,7 @@ AM_INIT_AUTOMAKE
 AC_PREREQ(2.59)
 AC_PREFIX_DEFAULT(/usr)
 AM_MAINTAINER_MODE
+AC_USE_SYSTEM_EXTENSIONS
 
 dnl *************************************************************
 dnl * Define the set of applicable options
-- 
2.0.4

